src/manifest.chrome.json

```diff
@@ -3,23 +3,24 @@
 
```

Comment:
You'll notice that there are manifest files for chrome and firefox. There are incompatibilities between what's supported / required between the two. Webpack picks the correct manifest at build-time.